test/validator_test.exs

```diff
@@ -5,19 +5,49 @@ defmodule ValidatorTest do
     test "checks that the user input is a non_numerical string" do
       user_input = "not a number"
 
-      assert Validator.validate(user_input) == {:error, :non_numerical}
+      board = %{
+        1 => :empty, 2 => :empty, 3 => :empty,
+        4 => :empty, 5 => :empty, 6 => :empty,
+        7 => :empty, 8 => :empty, 9 => :empty
+      }
+
+      assert Validator.validate(board, user_input) == {:error, :non_numerical}
```

Comment:
What do you think about returning an `invalid` atom instead of `error`?

Reply:
Hmmm, I don't have a strong preference either way at first glance, but I'd say since we do have a :valid atom for when the input is good, I think :invalid makes a little more sense in that context.

Reply:
Yea I'm leaning toward using `invalid` instead of `error`.